bigquery_etl/alchemer/response.schema.json

```
@@ -41,6 +41,11 @@
         "name": "type",
         "mode": "NULLABLE"
       },
+      {
+        "type": "STRING",
```

All the example `parent` values in [Alchemer's docs](https://apihelp.alchemer.com/help/surveyresponse-per-question-v5) are integers, so I'm pretty sure this would be better stored as an integer.
```suggestion
        "type": "INTEGER",
```

---

Yes, BigQuery will try to coerce types to the defined schema when loading data, but the load will fail with an error if such coercion fails.

It's unfortunate that the Alchemer API seems to be a bit inconsistent with how it encodes fields.  Their [docs](https://apihelp.alchemer.com/help/surveyresponse-returned-fields-v5#getobject) specify `survey_data[id]` as "numeric" but in your example it's being passed as a string, while `survey_data[section_id]` is also specified as "numeric" and in your example it's being passed as a number literal.

I'd still lean toward defining it as an integer field in BigQuery.